VehicleCode/TestingAndCalibration/SensorTesting/include/BrakeSensor.h

```diff
@@ -0,0 +1,59 @@
+#ifndef BRAKESENSOR_H
+#define BRAKESENSOR_H
+
+#include "Sensor.h"
+#include "SensorData.h"
+#include <Arduino.h>
+
+class BrakeSensor : public Sensor {
+    private:
+    int baseline;
+    int errorMargin;
+    bool pressState;
+    bool prevPressState;
+    bool driveState;
+    bool inError;
+    bool inCriticalError;
+    int brakeP;
+    int* sendData;
+    int dataLength;
+    int timeErrorStart;
+    int errorBaseline;
+    
+    FlexCAN_T4<CAN2, RX_SIZE_256, TX_SIZE_16> can2;
+    
+    void sendMotorCommand();
+
+
+
+    public:
+    BrakeSensor(int id, int waitTime, int inPin, int dataLength, int baseline, int errorMargin);
```

Comment:
this isn't necessary but I like to group function instantiations together by role/type